integration_tests/test_dbapi.py

```diff
@@ -171,12 +162,12 @@ def test_boolean_query_param(trino_connection):
     cur.execute("SELECT ?", params=(True,))
     rows = cur.fetchall()
 
-    assert rows[0][0] == True
+    assert rows[0][0]
```

Comment:
I wouldn't consider this equivalent. Can lead to silent bugs when copying tests.

This change is only safe because you know that the type is a boolean. Truthiness is different for other types.

Same at L170.